forge-gui/res/cardsfolder/upcoming/martha_jones.txt

```diff
@@ -0,0 +1,14 @@
+Name:Martha Jones
+ManaCost:2 U
+Types:Legendary Creature Human Cleric
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigInvestigate | TriggerDescription$ Woman Who Walked the Earth — When CARDNAME enters the battlefield, investigate. (Create a Clue token. It's an artifact with "{2}, Sacrifice this artifact: Draw a card.")
+SVar:TrigInvestigate:DB$ Investigate
+T:Mode$ Sacrificed | ValidPlayer$ You | ValidCard$ Clue | TriggerZones$ Battlefield | Execute$ DBUnblockable | TriggerDescription$ Whenever you sacrifice a Clue, CARDNAME and up to one other target creature can't be blocked this turn.
+SVar:DBUnblockable:DB$ Effect | RememberObjects$ Self | ForgetOnMoved$ Battlefield | StaticAbilities$ Unblockable | SubAbility$ TrigPump

```

Comment:
should still combine the effects by remembering both

Reply:
Is there an example script that does this that I could use as a reference?

Reply:
just do `RememberObjects$ Self,Targeted` on L9 and get rid of 8